### PR TITLE
React Insta-Clone

### PR DESCRIPTION
**Pull request by erin-koen**

### CODE REVIEW

File: instagram/src/App.js [hunk 1]
```diff
@@ -27,14 +27,17 @@ class App extends Component {
...
+    e.preventDefault();
+    if (this.state.filterElement === ''){
+      this.setState({ postData: dummyData })
+    } else {
+      let filteredResults = this.state.postData.filter(post => post.username.includes(this.state.filterElement))
+      this.setState({
+        postData: filteredResults
+      });
+    }
+  }
```
Comment (mikaelacurrier) [inline instagram/src/App.js]: HECK YES. so good.

File: instagram/src/App.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    console.log(this.state.postData)
+  }
+
+  shouldComponentUpdate(prevState) {
+    if (prevState.postData !== this.state.postData){
+      return true;
+    } else{
+      return false;
+    }
+  }
```
Comment (mikaelacurrier) [inline instagram/src/App.js]: Love it.

File: instagram/src/Components/CommentSection/CommentSection.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            username: this.state.username,
+            text: this.state.text,
+        }
+        console.log(newComment);
+        this.setState({
+            comments:[...this.state.comments, newComment],
+            text: '',
+        });
+    };
+
```
Comment (mikaelacurrier) [inline instagram/src/Components/CommentSection/CommentSection.js]: :raised_hands:

File: instagram/src/Components/Post Likes/PostLikes.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    addLike = e => {
+        e.preventDefault();
+        let newLikeTotal = this.state.likes + this.state.increment
+        console.log(newLikeTotal);
+        this.setState({
+            likes: newLikeTotal,
+            increment: 1,
+        })
+    }
+
```
Comment (mikaelacurrier) [inline instagram/src/Components/Post Likes/PostLikes.js]: yeah, I see what you're doing but I think the `++` would make your life a lot easier / code more readable

File: instagram/src/App.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+      <div className="app"> 
+        <PostPage />
+      </div>
+    );
+  }
+
+    
+}
+
+export default authenticate(App)(Login);
```
Comment (mikaelacurrier) [inline instagram/src/App.js]: so close to perfect - try to capitalize all your components, even the HOCs

Comment (erin-koen) [reply instagram/src/App.js]: yeah you know - i thought this too. But the readme example was persistently lowercase... so i thought it was some syntax thing i missed where HOCs were lowercase.

**Pull request merged**